CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add use of runtime/default seccomp profile.
 - Add toleration for new control-plane taint.
 
+### Changed
+
+- Update `kubectl` image to `1.23.15` and align CRD install values with other providers.
```

Comment:
We should probably advertise this as breaking change since there is a change in the schema.